packages/aws/data_stream/apigateway_logs/manifest.yml

```
@@ -85,7 +85,13 @@ streams:
       - name: max_number_of_messages
         type: integer
         title: "[SQS] Maximum Concurrent SQS Messages"
-        description: The maximum number of SQS messages that can be inflight at any time.
+        description: >
+          The maximum number of SQS messages that can be inflight at any time. Defaults to 5. When processing large 
+          amount of large size S3 objects and each object has large amount of events, if this parameter sets too high,
+          it can cause the input to process too many messages concurrently, overload the agent and cause ingest failure.
+          We recommend to keep the default value 5 and use the {fleet-guide}/es-output-settings.html#es-output-settings-performance-tuning-settings[preset]
```

You should hard-code the full link instead of using the AsciiDoc attribute (`{fleet-guide}`) here. I'm not sure if the manifest descriptions support Markdown formatting (an integration developer may know), but if it does you can use this:

```suggestion
          We recommend to keep the default value 5 and use the [preset](https://www.elastic.co/guide/en/fleet/current/es-output-settings.html#es-output-settings-performance-tuning-settings)
```

---

Will do! Thank you! I was originally worried about the `/current/` version in the link. But since we are using it in the integrations not beats, we should be good 😂